### DISCUSSION TITLE
could the sandbox protect me from some applications?

### DISCUSSION THREAD

**Discussion by miaomiaolong** [sandboxie-plus/Sandboxie#2126] [Q&A Feedback]
it's know that some applications read or is reading our personal data on a laptop, such as qq and wechat. God knows what they are doing. so i am asking whether sandbox could stop them from doing so, or just feed them a fake or empty directory, or anything else, thus protect my privacy.  and how i can enable this feature.

**Comment by miaomiaolong** [comment]
ok, i think i know how to do it. (https://nanzt.info/8982.html)

**Comment by bastik-1001** [comment]
You could use` UsePrivacyMode=y` for those boxes that should not have access to user data. This setting restricts a sandbox's read access to generic system folders like C:\Windows and C:\Program Files. This setting requires a supporter certificate.

And as you found out, you can restrict access to folders per `ClosedFilePath`.

Edit: Here are the documentation resources for those settings. [UsePrivacyMode](https://sandboxie-plus.github.io/sandboxie-docs/Content/PrivacyMode.html) and [ClosedFilePath](https://sandboxie-plus.github.io/sandboxie-docs/Content/ClosedFilePath.html) as well an [overview over the resource access settings](https://sandboxie-plus.github.io/sandboxie-docs/Content/ResourceAccess.html)

**Comment by BackMountainDevil** [comment]
I am trying to use Sandboxie to run wechat. There are some errors info. But those errs not make an influence to wechat. Have you meet that?

**Reply by BackMountainDevil** [reply]
And I tried install QQ but failed.
